src/commonMain/kotlin/baaahs/plugin/beatlink/TrackData.kt

```diff
@@ -0,0 +1,10 @@
+package baaahs.plugin.beatlink
+
+data class TrackData(
+    val id: Int,
+    val title: String,
+    val artist: String,
+//    val art: BufferedImage?,
+    val key: String,
+    val cueList: List<Long>,
```

Comment:
You won't be able to pass variable sized arrays to GLSL. You could make this a size-8 array of float timestamp values, since almost all DJs only use cue points A-H.